src/signals/incident-management/containers/IncidentDetail/components/MetaList/index.test.js

```diff
@@ -203,17 +194,18 @@ describe('MetaList', () => {
 
     it('should be visible', () => {
       render(renderWithContext());
-
       expect(screen.getByText(subcategoryLabel)).toBeInTheDocument();
-      expect(screen.getByText(parentIncident.category.sub)).toBeInTheDocument();
+      const re = new RegExp(`${parentIncident.category.sub}`, 'g');
```

Comment:
Why does this need to be a regular expression?

Reply:
The generation of the display values for the categories contains at the end a list of the departments. I have chosen not to test that that was build and keep the unittest changes minimal. I could add that extra test if you think is necessary.

Reply:
I don't understand. You can still test it with a string instead of a regular expression, right?

Reply:
Of course, I'll use a static string then, thanks!